phpstan.neon.dist

```diff
@@ -3,4 +3,11 @@ parameters:
 
 	paths:
 		- app
-		- domain/src
\ No newline at end of file
+		- domain/src
+
+	checkMissingCallableSignature: true
+
+	type_coverage:
+		return_type: 98
+		param_type: 98
```

Comment:
ℹ️ Can't go 100% here because there are some missing return and parameter types but the rule currently won't tell me where exactly (see this [issue](https://github.com/TomasVotruba/type-coverage/issues/9))